internal/services/users/user_resource.go

```diff
@@ -195,6 +195,12 @@ func userResource() *schema.Resource {
 				Computed:    true,
 				Description: "The primary cellular telephone number for the user.",
 			},
+			"user_type": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
```

Comment:
should we default to member instead of computed here?

Reply:
This field isn't mandatory in api, I won't override default behaviour set by Microsoft if user set no value.
Is ok for your ?

Reply:
I think it would be more predictable to have a default here - guest users should probably be explicitly configured as such and the intention is clearer if we default to a regular user.